services/initChatAnonymous/initChatAnonymousServices.ts

```diff
@@ -52,15 +52,15 @@ type InitChatAnonymousPayload = {
   anon_user_info_color_name: string;
   anon_user_info_emoji_code: string;
   anon_user_info_emoji_name: string;
-  members: string[];
+  member: string;
   message: string;
 };
 
 const initChatAnonymous = async (
   payload: InitChatAnonymousPayload
 ): Promise<InitChatAnonymousResponse> => {
   const { data } = await apiAnonymous.post<AxiosResponse<InitChatAnonymousResponse>>(
-    "/api/v1/chat/init-chat-anonymous",
+    "/api/v1/chat/init-chat-anonymous-v2",
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `members` to `member` in the response type might cause issues if the backend still returns an array of members. If the backend has been updated to return a single member, then this change is fine. However, if not, you will need to handle the array of members correctly.

If the backend hasn't been changed and still returns an array, you can revert the change:

```diff
-  member: string;
+  members: string[];
```

Also, I noticed that the endpoint has been changed from `/api/v1/chat/init-chat-anonymous` to `/api/v1/chat/init-chat-anonymous-v2`. Please ensure that the new endpoint is live and working as expected before merging these changes.


<!-- This is an auto-generated comment by OSS CodeRabbit -->